tests/unit/test_views.py

```diff
@@ -147,6 +147,19 @@ def test_logged_out_redirects_login(self):
             "/accounts/login/?next=/foo/bar/%3Fb%3Ds"
 
 
+class TestForbiddenIncludeView:
+
+    def test_forbidden_include(self):
+        exc = pretend.stub()
+        request = pretend.stub()
+
+        resp = forbidden_include(exc, request)
+
+        assert resp.status_code == 200
```

Comment:
Do we still want this to be a ``403`` response? I think the HTML include will work fine in either case, but it feels a little cleaner to keep the response code as ``403`` here? What do you think?